build.xml

```diff
@@ -209,8 +209,12 @@
     <fail unless="test.mongodb-database" message="test.mongodb-database not set"/>
     <fail unless="test.admin" message="test.admin not set"/>
     <fail unless="test.admin-pwd" message="test.admin-pwd not set"/>
+    <fail unless="test.admin-token" message="test.usr1-token not set"/>
```

Comment:
although test.admin is actually required because it sets the admin username for the service, so I'm leaving that.